### PR TITLE
sys-devel/llvm: cleanup of old versions

### PR DESCRIPTION
**Pull request by mgorny**

### CODE REVIEW

**Comment by mgorny** [comment]
@gentoo/prefix, could you check if the simplified 3.5+3.6 ebuilds are suitable for your bootstrap problem?

**Comment by mgorny** [comment]
(I've tested that they both build and install fine on Linux)

File: sys-devel/llvm/llvm-3.4.2-r100.ebuild [hunk 1]
```diff
@@ -203,8 +197,8 @@ src_install() {
 	if ! use clang; then
 		rm "${WORKDIR}"/${PN}-3.4-manpages/clang.1 || die
 	else
-		for tool in clang{,++}{,-${PV}} ; do
-			dosym /usr/bin/${tool} /usr/bin/${CHOST}-${tool}
+		for tool in clang{,++} ; do
+			dosym ${tool} /usr/bin/${CHOST}-${tool}
```
Comment (grobian) [inline sys-devel/llvm/llvm-3.4.2-r100.ebuild]: ${tool} doesn't include path, and I don't see a cd or pushd, so does this work as anticipated?

I guess you wanted to use something like:
`dosym ${EPREFIX}/usr/bin/${tool} /usr/bin/${CHOST}-${tool}`
(as per bug #615594)?

Comment (mgorny) [inline sys-devel/llvm/llvm-3.4.2-r100.ebuild]: It's a relative symlink. Using absolute path for a symlink to the file in the same directory is just stupid.

Comment (grobian) [inline sys-devel/llvm/llvm-3.4.2-r100.ebuild]: ok, assuming it just works then

**Pull request closed without merging**